admin-web/src/main/java/hu/schonherz/admin/web/locale/LocalizationManagement.java

```diff
@@ -0,0 +1,68 @@
+package hu.schonherz.admin.web.locale;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+import javafx.beans.property.ObjectProperty;
```

Comment:
http://stackoverflow.com/questions/4830588/localization-in-jsf-how-to-remember-selected-locale-per-session-instead-of-per

Ezt légyszives csináld meg BalusC commentje alapján, mivel ez így 1 felhasználóra igaz maximum. (static adattagok, static metódosok miatt több felhasználó locale-je egyszerre változna)